Makefile

```diff
@@ -142,21 +142,22 @@ verify-vendor:
 generate-kustomize: bin/helm
 	cd charts/aws-ebs-csi-driver && ../../bin/helm template kustomize . -s templates/clusterrole-attacher.yaml > ../../deploy/kubernetes/base/clusterrole-attacher.yaml
 	cd charts/aws-ebs-csi-driver && ../../bin/helm template kustomize . -s templates/clusterrole-provisioner.yaml > ../../deploy/kubernetes/base/clusterrole-provisioner.yaml
-	cd charts/aws-ebs-csi-driver && ../../bin/helm template kustomize . -s templates/clusterrolebinding-attacher.yaml > ../../deploy/kubernetes/base/clusterrolebinding-attacher.yaml
-	cd charts/aws-ebs-csi-driver && ../../bin/helm template kustomize . -s templates/clusterrolebinding-provisioner.yaml > ../../deploy/kubernetes/base/clusterrolebinding-provisioner.yaml
+	cd charts/aws-ebs-csi-driver && ../../bin/helm template kustomize . -s templates/clusterrole-resizer.yaml -f ../../deploy/kubernetes/values/resizer.yaml > ../../deploy/kubernetes/overlays/alpha/clusterrole-resizer.yaml
```

Comment:
should we move resizer out of the alpha overlay along with snapshotter? Then we can delete the alpha overlay

Reply:
I certainly can.  Is it out of alpha?  Moving them is easy enough as long as functionally it is okay to do so.  I assume that also means removing the boolean that turns it on and off?

Reply:
Yes, it's out of alpha since 1.16, and we recently stopped supporting <=1.16 clusters so now is a good time to remove. ref: https://kubernetes-csi.github.io/docs/volume-expansion.html. 

So yes, we can deprecate enableVolumeResizing and default it true and eventually remove it. To be consistent we should do the same for enableVolumeScheduling and enableVolumeSnapshot. All these flags represent features that were once alpha but have since matured to Beta/GA, they aren't really needed anymore.

that said, I can handle deprecating enableVolumeScheduling enableVolumeSnapshot and enableVolumeResizing in a follow-up PR, don't want to expand the scope of this PR too much.

Reply:
I will move the resizer and scheduler out of alpha, and with that, the enableVolumeResizing and enableVolumeScheduling will no longer be needed so I will just remove them.  I am leaving enableVolumeSnapshot for now but only as a flag on the but only as a gate for the snapshot controller stateful set since the thought right now is to not change how that works.